examples/rapids_simple.py

```diff
@@ -0,0 +1,63 @@
+"""
+Optuna example using RAPIDS library for optimization.
+
+In this example, we perform HPO on Iris dataset using cuML's
+RandomForestClassifier. This should be used a starting point
```

Comment:
I'm not quite sure but this should be rather...
```suggestion
RandomForestClassifier. This should be used as a starting point
```